### PR TITLE
Mozhdeh's homework

### PR DESCRIPTION
**Pull request by mozhdeh-ahmadi**
🚨 REMEMBER

1️⃣ We will review together the work you have done so far. 

2️⃣ Add my name in the Reviewers section in the right sidebar 👉👉👉

3️⃣ Answer these reflection questions before each PR:

  * What was most challenging about the homework so far? 
using command in terminal

  * How did you solve it?
I asked my friends and searched in google.

### CODE REVIEW

**Review by nikosolihin — CHANGES_REQUESTED** [review]
@mozhdeh-ahmadi I can't review anything since all your files are empty.

**Comment by mozhdeh-ahmadi** [comment]
I have finished it! @nikosolihin

File: forking.md [hunk 1]
```diff
@@ -0,0 +1 @@
+https://github.com/mozhdeh-ahmadi/best_animals
```
Comment (nikosolihin) [inline forking.md:1]: Your `birds.txt` and `fish.txt` files are still empty. Once you push some changes to the repo, I'll mark this as correct.

File: googling.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# question number one
+  
+  
+  Phrase 1: how can change a typo in commit messages?
+
+  Phrase 2: how can i change a commit messages before I push it to github?
+
+  # question number two
+   Phrase 1: what are the Ruby error  messages with meaning?
```
Comment (nikosolihin) [inline googling.md:9]: This is quite generic for a Google phrase. Something like `ruby unterminated string meets end of file` will probably get you closer to the relevant results.

File: googling.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+  Phrase 2: how can i fix a string error message in Ruby program?
```
Comment (nikosolihin) [inline googling.md]: Same as my comment above — you need the actual error message.

File: googling.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+  # question number three
+
+ Phrase 1: how can I print a git log  in oneline?
+   
+ Phrase 2: how can i git log in a mod 0?
```
Comment (nikosolihin) [inline googling.md]: I'd say the mod0 program is irrelevant in this case. You'd be better off including the keyword `git` like `git log in oneline`.

File: terminal.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+cd ..
+
+## If I'm in the projects_to_delegate directory, what two commands do I need to type to get to to_do?(We'll learn how to combine these momentarily)
+
+cd  ..
+cd ..
+## I'm in the home directory. What three commands do I need to type to get to projects_to_delegate?(We'll learn how to combine these momentarily)
+
+cd  cd  work/projects-delegate
```
Comment (nikosolihin) [inline terminal.md:11]: Did you mean `cd ..`

File: terminal.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+## I'm in the projects_to_delegate directory. What three commands do I need to type to get to home?(We'll learn how to combine these momentarily)
+
+cd ..
+cd ..
+cd ..
```
Comment (nikosolihin) [inline terminal.md:16]: This should actually be `cd home`

File: terminal.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+cd  ..  rm -rf  home
+
+### I'm in the projects_to_delegate directory. What two commands do I type to remove the directory I'm currently in?
+
+cd .. rm -rf  project-to-delegate
+
+### I'm in projects_to_delegate. What four commands do I need to type to remove the cleaning.txt file?
+
+cd  .. cd ..  cd rm  cleaning.txt
```
Comment (nikosolihin) [inline terminal.md]: You miss `cd home` before removing cleaning.txt

**Review by nikosolihin — CHANGES_REQUESTED** [review]
@mozhdeh-ahmadi Please make these corrections before we merge to master today.

**Pull request merged**